ls1intum/ArTEMiS
# Setup/Add cold dev sever

### Checklist

### Motivation and Context
The HMR for Angular is not working correctly for us. Instead of replacing single chunks, it always reloads the page.
I did some investigation following up @maximilianmeier but found no solution.

As a compromise I have created a new script that sets the hot & inline params of the webpack dev server to false. This way the page has to be manually reloaded - as the HMR is not working, it is a better approach imo.

### Description
- New script start-cold in package.json
- Added checking for hot parameter in webpack.dev.js

### Steps for Testing


1. Start client with start-cold
2. Change a file
3. Modules should recompile but page should not reload
4. Reload the page and the change should be visible

### Screenshots
